secureli/abstractions/pre_commit.py

```diff
@@ -762,3 +784,71 @@ def _compare_repo_versions(self, current_config: dict, expected_config: dict):
         )
 
         return output
+
+    def _load_language_config_file(self, language: str) -> LoadLanguageConfigsResult:
```

Comment:
These new functions don't have any unit tests.  Please them to test_prec_ommit.py